src/proxy.ts

```diff
@@ -49,4 +49,30 @@ function computeApi3ReaderProxyV1Address(
   );
 }
 
-export { computeApi3ReaderProxyV1Address };
+function computeCommunalApi3ReaderProxyV1Address(chainId: ethers.BigNumberish, dapiName: string) {
```

Comment:
"Communal" sounds weird, but I can't really think of any word except public, but that implies something is private which isn't necessarily true

Reply:
It's not a commonly used word so it works well as a term. It does have some negative connotations but we don't want people to use the communal proxy anyway.